src/main/java/com/regnosys/rosetta/common/serialisation/RosettaXMLObjectMapperCreator.java

```diff
@@ -0,0 +1,17 @@
+package com.regnosys.rosetta.common.serialisation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.regnosys.rosetta.common.serialisation.mixin.RosettaXMLModule;
+import com.rosetta.util.serialisation.RosettaXMLConfiguration;
+
+public class RosettaXMLObjectMapperCreator extends AbstractRosettaObjectMapperCreator {
```

Comment:
I would use composition here instead of inherence. It is neater and more easily testable.